cmd/grpcurl/grpcurl.go

```diff
@@ -54,6 +54,9 @@ var (
     	read from stdin. For calls that accept a stream of requests, the
     	contents should include all such request messages concatenated together
     	(optionally separated by whitespace).`)
+	reflectWithHeaders = flag.Bool("reflect-with-headers", false,
+		`Include headers provided with -H when performing reflection calls
+		against a remote server.`)
```

Comment:
Do you mind double-checking how this looks when you do `grpcurl -h`? I remember the formatting of flag help in the `flag` package is not great, so I had to manually format the other remarks in here so they rendered properly. For example, this second line may need lower level of indent, like the `data` parameter above.

Reply:
That's correct. I'll commit the fixed indentation as soon as we settle on a solution for the other issue.

Reply:
FYI, in Go 1.10 there's [an improvement](https://golang.org/doc/go1.10#flag) that provides better alignment of flag usage strings when including newlines. The current output on 1.10 is therefore funky again for all flags.